nemo/collections/llm/api.py

```diff
@@ -203,6 +204,8 @@ def finetune(
         optim (Optional[OptimizerModule]): The optimizer module to be used. If not provided, the default
             optimizer from the model will be used.
         peft (Optional[PEFT]): A PEFT (Parameter-Efficient Fine-Tuning) configuration to be applied.
+        tokenizer (Optional[TokenizerType]): Tokenizer setting to be applied. Can be 'data' or 'model'
+            or an instance of TokenizerSpec.
```

Comment:
perhaps you can explain the reason to specify a new tokenizer rather than using the one that comes with the model here in the docstring